src/ui/actions/app.ts

```diff
@@ -20,7 +20,7 @@ import { openQuickOpen } from "devtools/client/debugger/src/actions/quick-open";
 import { getRecordingId } from "ui/utils/recording";
 import { prefs } from "devtools/client/debugger/src/utils/prefs";
 import { shallowEqual } from "devtools/client/debugger/src/utils/resource/compare";
-import type { ThreadFront as ThreadFrontType } from "protocol/thread";
+import { ThreadFront as ThreadFrontType } from "protocol/thread";
```

Comment:
:hand: Any reason this got changed to remove the `import type` ? I specifically wanted to avoid directly importing `ThreadFront` here if possible.

Reply:
Yep, I accidentally imported it, then realized I had ruined your work, and tried to reverse my mistake, but I must have not reverted the actual import! Sorry about that, this will not ship with this change for sure!